src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Snackbar/SnackBar.tizen.cs

```diff
@@ -27,7 +27,10 @@ internal void Show(Forms.Page sender, SnackBarOptions arguments)
 				ok.Clicked += async (s, evt) =>
 				{
 					snackBarDialog.Dismiss();
-					await action.Action();
+
+					if (action?.Action != null)
```

Comment:
`action` is not null here

```suggestion
					if (action.Action != null)
```